src/test/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationControllerTest.java

```diff
@@ -71,6 +82,10 @@ void setup() {
     @DisplayName("Get View Confirmation Screen - success path")
     void getRequestSuccess() throws Exception {
 
+        when(mockCompanyService.getCompanyProfile(COMPANY_NUMBER))
+                .thenReturn(PPSTestUtility.validCompanyProfile(COMPANY_NUMBER));
+        when(mockPayablePenaltyService.getPayableLateFilingPenalty(COMPANY_NUMBER, PENALTY_ID))
```

Comment:
`PENALTY_ID` should probably be renamed to be `PENALTY_NUMBER` just for consistency with the path params / naming already throughout the code?

(Although we're calling it "Penalty reference" these days :))

Reply:
Actually NO... Think we need to be careful here with this path param and our naming. This isn't actually the Penalty reference by the looks of it, it is the Payment reference.

You’ll notice in the address bar shows that it has been paid with a payment reference number e.g. DJ89040912. (`http://chs.local/late-filing-penalty/company/NI038379/penalty/DJ89040912/confirmation?ref=late_filing_penalty_DJ89040912&state=2fb58c76-8b9b-4773-8bb6-04686c6596e7&status=paid`)
You can use Studio 3T to connect to mongodb://127.0.0.1:27017 and find the Mongo Documents in the payments > payments collection using `"state" : "2fb58c76-8b9b-4773-8bb6-04686c6596e7"` and  late_filing_penalties > payable_resources `"reference" : "DJ89040912"`

Reply:
I have changed that to penaltyNumber  and PENALTY_NUMBER in testing part, because other pages are using penaltyNumber

Reply:
Sorry @klai-ch, I think I convinced myself the other day that it must intentionally have been different on the confirmation page, wrt the `ConfirmationController` path param not having `penaltyNumber`?
Does my second comment make sense to you, as when I looked the other day, the confirmation screen showed a different ref e.g. `Penalty reference: NQ72384556`

```
-@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyId}/confirmation")
+@RequestMapping("/late-filing-penalty/company/{companyNumber}/penalty/{penaltyNumber}/confirmation")
```

Reply:
After discussion, we will have another ticket SAN-177 for setting up the page using penaltyReference instead of TransactionReference, currently keep using TransactionReference as penaltyId at this moment